front/src/App.js

```diff
@@ -168,13 +194,7 @@ class App extends Component {
       <ThemeProvider theme={theme}>
         {loggedIn ? (
           <Container>
-            {activePage === "refbox" && (
-              <div>
-                <RefboxField isPenalty={false} token={this.state.token} />
-                <RefboxSettings token={this.state.token} />
-              </div>
-            )}
-            {activePage === "settings" && (
+            {activePage === pageTypes.SETTINGS && (
               <Fragment>
```

Comment:
Consider adding classname={activePage === pageTypes.SETTINGS? "":"hide-xs hide-sm"} here instead of making this entire component optional.